apps/www/_blog/2022-10-05-supabase-beta-update-september-2022.mdx

```diff
@@ -45,8 +41,8 @@ We launched the new Auth UI on Product Hunt! Auth UI is a pre-built React compon
 
 If you've ever wondered, "can I run Postgres inside a browser, using an embeddable Linux Virtual Machine?", wonder no longer. With our friends at [Snaplet](https://www.snaplet.dev/?utm_campaign=Beta%20update%20emails&utm_source=hs_email&utm_medium=email&_hsenc=p2ANqtz-8u0x9grbiydYqh56hC_8Dta4ZiTdWbW0eXWkVCnUEUGWWoVDFXCpWzJWImkLfM-lG-o2JX), we've released an open source Postgres WASM.
 
-[Read the blog post](https://supabase.com/blog/postgres-wasm) <br/>
-[Comment on Hacker News](https://news.ycombinator.com/item?id=33067962) <br/>
+[Read the blog post](https://supabase.com/blog/postgres-wasm)
+[Comment on Hacker News](https://news.ycombinator.com/item?id=33067962)
 Visit the repo: [Snaplet](https://github.com/snaplet/postgres-wasm) | [Supabase](https://github.com/supabase-community/postgres-wasm)
```

Comment:
I think these will end up all on one line without the line breaks and if it's not a list.

Reply:
good spot! reverted